### DISCUSSION TITLE
whatsapp_business_management

### DISCUSSION THREAD

**Discussion by luisdanielhj** [chatwoot/chatwoot#5940] [Self hosted] [answered]
Do you have any advice or a test video to get this permission from Facebook?

whatsapp_business_management

**Comment by sojan-official** [comment, marked as answer]
@luisdanielhj  could you give more details or add a screencast of what you are trying to achieve? 
for configuring WhatsApp channel you don't need to get permissions from facebook 
ref: https://www.chatwoot.com/docs/product/channels/whatsapp/whatsapp-cloud/
